internal/provider/framework/config.go

```diff
@@ -334,6 +334,7 @@ func (p *ProviderConfig) Load(ctx context.Context, data *ProviderModel, tfVersio
 				f.VirtualMachine.DeleteOSDiskOnDeletion = feature[0].DeleteOsDiskOnDeletion.ValueBool()
 			}
 
+			// TODO: Remove in 5.0
```

Comment:
This does nothing now and defaults to `false` anyway, so I think we can just remove it?

Reply:
Fixed.